src/lib/import-export/tests/export/exporters/default-exporter.test.ts

```diff
@@ -0,0 +1,201 @@
+import fs from 'fs';
+import fsPromises from 'fs/promises';
+import os from 'os';
+import archiver from 'archiver';
+import { DefaultExporter } from '../../../export/exporters/default-exporter';
+import { ExportOptions, BackupContents } from '../../../export/types';
+
+jest.mock( 'fs' );
+jest.mock( 'fs/promises' );
+jest.mock( 'os' );
+
+// Create a partial mock of the Archiver interface
+type PartialArchiver = Pick<
+	archiver.Archiver,
+	'pipe' | 'file' | 'directory' | 'finalize' | 'on' | 'abort'
+>;
+
+const createMockArchiver = (): jest.Mocked< PartialArchiver > => {
+	return {
+		pipe: jest.fn().mockReturnThis(),
+		file: jest.fn().mockReturnThis(),
+		directory: jest.fn().mockReturnThis(),
+		finalize: jest.fn().mockResolvedValue( undefined ),
+		on: jest.fn().mockReturnThis(),
+		abort: jest.fn(),
+	};
+};
+
+// Mock archiver module
+jest.mock( 'archiver', () => {
+	return jest.fn().mockImplementation( () => createMockArchiver() );
```

Comment:
This could be simplified with:
```suggestion
	return jest.fn( () => createMockArchiver() );
```